plugins/source/azure/client/services/container.go

```diff
@@ -1,16 +1,18 @@
-//go:generate mockgen -destination=./mocks/containerregistry.go -package=mocks . ContainerRegistriesClient,ContainerReplicationsClient
+//go:generate mockgen -destination=./mocks/containerregistry.go -package=mocks . ContainerRegistriesClient,ContainerReplicationsClient,ContainerManagedClustersClient
```

Comment:
Instead of `containerregistry.go` and `containerservice.go` I put both in the same `container.go` file